internal/datastore/mysql/datastore_test.go

```diff
@@ -340,14 +326,12 @@ func GarbageCollectionByTimeTest(t *testing.T, ds datastore.Datastore) {
 	}
 	relationship := tuple.ToRelationship(tpl)
 
-	relLastWriteAt, err := ds.WriteTuples(
-		ctx,
-		nil,
-		[]*v1.RelationshipUpdate{{
+	relLastWriteAt, err := ds.ReadWriteTx(ctx, func(ctx context.Context, rwt datastore.ReadWriteTransaction) error {
```

Comment:
Maybe we should have a datastore util function for writing a set of rels, without the syntax around opening the transaction, then calling WriteRelationships?

Reply:
It's really only an issue for tests, and I already moved them all over. I don't think we should hide the way it works under the covers.